openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -474,7 +474,8 @@
     "no-of-test": " No. of Test",
     "test-suite": "Test Suite",
     "enter-entity": "Enter {{entity}}",
-    "test-suite-status": "Test Suite Status"
+    "test-suite-status": "Test Suite Status",
+    "mlmodel": "Mlmodel"
```

Comment:
I think It should be "ML Model"

Reply:
Thanks for pointing that out! and Yes it should go that way!